### DISCUSSION TITLE
Sqlite instead of MySQL

### DISCUSSION THREAD

**Discussion by mitulmandaliya** [rahulvemula/SE-leasingPortal#50] [Ideas]
I am thinking to use sqlite instead of MySQL so all of us don't have to install MySQL server on local machine. I found db4free.net which provides free MySQL hosting up to 200 MB but I couldn't find much reviews on how reliable it is.

**Comment by vamsibvv** [comment]
Can Sqlite handle complex operations for our db? And do we have enough online community for resolving db issues.

Also, do we need to install any workbench in our local systems to test our application?

**Comment by mitulmandaliya** [comment]
Yes it can handle all operations like any relational database. No we don't have to workbench but we can install DB Browser which is sqlite client.
